fn_digital_shadows_search/tox.ini

```diff
@@ -12,7 +12,7 @@ skip_missing_interpreters=True
 passenv=TEST_RESILIENT_*
 commands = pytest -s {posargs}
 deps =
-    pytest
+    pytest==4.0.1
```

Comment:
This may have been checked in by mistake? Should not be editing file in `digital_shadows` directory